requirements.txt

```diff
@@ -0,0 +1,8 @@
+numpy
+scipy
+pandas>=0.23
+plotly>=4.7.0
+statsmodels
+sklearn
+keras
```

Comment:
Do we really need keras? I thought they were now part of the tensorflow package.

Reply:
There's only a single import from keras. But it seems to be possible to remove it